apis/token-price/lib/api/v2.ts

```diff
@@ -64,28 +60,24 @@ interface PoolCode {
   poolName: string
 }
 
-const REDIS_KEY_PREFIX = 'token-list-v2-'
-
-async function fetchTokensFromLists() {
-  const promises: Promise<TokenList>[] = []
-
-  for (const url of DEFAULT_LIST_OF_LISTS) {
-    const key = `${REDIS_KEY_PREFIX}-${url}`.toLowerCase()
-    const cached = await redis.get(key)
-    if (cached) {
-      promises.push(Promise.resolve(JSON.parse(cached)))
-    }
-  }
+async function fetchTokens(chainId: ExtractorSupportedChainId) {
+  const result = await fetch(
+    `https://tokens-git-feature-token-v2-api.sushi.com/api/v1/${chainId}/`,
+  )
+  // const result = await fetch(`https://tokens.sushi.com/v1/${chainId}/`)
```

Comment:
I will change these urls in another PR after, they need to be merged to master first.

Reply:
Maybe it'd make sense to move the URL to an env var? So that we don't have to deal with this next time

Reply:
Nah, then we'll have an extra env just laying there, I'd rather just pr and fix it, it's a one time issue